tests/unit/drivers/test_segmenter_driver.py

```diff
@@ -60,7 +59,7 @@ def test_segment_driver():
     assert valid_doc.chunks[1].blob == array2pb(np.array([1.0, 1.0, 1.0]))
     assert valid_doc.chunks[1].weight == 1
     assert valid_doc.chunks[1].length == 3
-    assert valid_doc.chunks[1].mime_type == "image/png"
+    assert valid_doc.chunks[1].mime_type == ""
```

Comment:
Sorry for the confusion, but: Could you make this again "image/png"? I think it demonstrates better how the inheritance of this attribute works.